generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -206,17 +206,10 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
         <%_ }) _%>
       <%_ } _%>
 
-      const cloneEntity = Object.assign({}, entity);
-      for (const property in entity) {
-        if (entity[property] === '') {
-          cloneEntity[property] = null;
-        }
-      }
-
       if (isNew) {
-        props.createEntity(cloneEntity);
+        props.createEntity(entity);
```

Comment:
the ticket https://github.com/jhipster/generator-jhipster/issues/12784 continue to work ? if yes, you can remove `cloneEntity` above

Reply:
Yes it keeps working (Tried with oauth2 and keycloak, and tried before and after your fix).
Is there still a `cloneEntity` that I didn't remove ?

Reply:
nope it's good :)